src/apis/drive/samples/node/README.md

```diff
@@ -0,0 +1,26 @@
+# Drive v3 API Samples
```

Comment:
There's a 0% chance we maintain a readme for this stuff :) If this is auto-generated, fine.  But we can't hand write this stuff.

Reply:
But why? This one was hand-written, I just cropped it. We only have samples for several APIs, we are not going to have more `README` files than number of hand-written samples we support.

Reply:
It's going to get out of date.  And without tooling to generate it, there's no way we keep up with them.  I prefer to have no README at all in the samples dir, and put whatever info we need to run the samples up in the top level README.